sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -858,3 +860,60 @@ def merge_only(self, accumulators):
 
   def extract_only(self, accumulator):
     return self.combine_fn.extract_output(accumulator)
+
+
+class Latest(object):
+  """Combiners for computing the latest element"""
+
+  class Globally(ptransform.PTransform):
+    """Compute the element with the latest timestamp from a
+    PCollection."""
+
+    @staticmethod
+    def add_timestamp(element, timestamp=core.DoFn.TimestampParam):
+      return [(element, timestamp)]
+
+    def expand(self, pcoll):
+      return (pcoll
+              | core.ParDo(self.add_timestamp)
+              | core.CombineGlobally(LatestCombineFn()))
+
+  class PerKey(ptransform.PTransform):
+    """Compute elements with the latest timestamp for each key
+    from a keyed PCollection"""
+
+    @staticmethod
+    def add_timestamp(element, timestamp=core.DoFn.TimestampParam):
+      _check_instance_type(KV[T, T], element)
+      (K, V) = element
```

Comment:
Maybe consider changing these local variables to (k, v) since K, V are type variables already defined.

Reply:
Or in a similar vein to element, timestamp etc, use key, value = element. Also I believe you do not need the paranthesis on the left.

Reply:
Thank You @robinyqiu and @aaltay. I'll make the change.